### PR TITLE
Add contact form section portfolio

### PR DESCRIPTION
**Pull request by ONJoseph**
-Add contact form section html css codes
-Add background image
-Fix indentation
-Add contact title and paragraph text
-Modified style.css file
-Modified README to reflect project progress

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -211,6 +211,33 @@ <h3 class="about__skills__top__title fw-500 fz-20 lh-24 clr-n-900">Skills</h3>
...
+				  </li>
+				  <li>
+					<input type="email" class="contact__email fw-500 fz-17 lh-20" placeholder="Enter your email" name="contact_email" required>
+				</li>
+				<li>
+					<textarea class="fw-500 fz-17 lh-20" name="contact_message" placeholder="Write your message here" cols="30" rows="8" maxlength="500" required></textarea>
+				</li>
+				<li>
+					<input type="submit" value="Get It Touch" class="btn fw-500 fz-17 lh-24">
+				</li>
```
Comment (ShadyShawkat) [inline index.html]: - [x] [OPTIONAL] Kindly change this button from saying 'Get It Touch' to 'Get In Touch', it's a typo in the Figma design.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @ONJoseph, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ONJoseph) [reply index.html]: Thanks immensely for the uplifting review! Will implement the optional suggestion.

Comment (ONJoseph) [reply index.html]: Done.

Comment (ShadyShawkat) [reply index.html]: Thank you.

**Pull request merged**